data/campaigns/The_Rise_Of_Wesnoth/scenarios/08_Clearwater_Port.cfg

```diff
@@ -40,7 +41,8 @@
         profile="portraits/aethyr-first.png"
         canrecruit=yes
         recruit=Peasant, Bowman, Horseman, Mage, Spearman
-        {GOLD 220 180 140}
+        {GOLD 320 220 120}
```

Comment:
I think this is too much gold, if you really want you can give them a bit more income, but keep the gold value as the original please. Same thing with side 1, don't give the player so much gold, but a slight bit more income is fine. Slight means something like 6 4 2.

In general this scenario is not supposed to be too easy, so I'd rather not over simplify it by giving the allied sides so much gold.

Reply:
Note that this was my original diff: https://github.com/cooljeanius/wesnoth_mods/blob/master/campaigns/The_Rise_Of_Wesnoth/diffs/08_Clearwater_Port.cfg.diff
Here's the replay: 
[TRoW-Clearwater Port replay.gz](https://github.com/wesnoth/wesnoth/files/6133760/TRoW-Clearwater.Port.replay.gz)

Reply:
btw @Dalas121 is considering further changes to these values in #7525